Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -1410,6 +1410,16 @@ void CGame::QuitPlayer(CPlayer& Player, CClient::eQuitReasons Reason, bool bSayI
     // If he had joined
     if (Player.IsJoined())
     {
+        // Check if the player was inside a team and fire the event
```

Comment:
I'd name the event in the comment to remove the ambiguity
```suggestion
        // Check if the player was inside a team and fire onPlayerTeamChange event
```